cpg_workflows/inputs.py

```diff
@@ -107,7 +107,7 @@ def _populate_cohort(cohort: Cohort, sgs_by_dataset_for_cohort, read_pedigree: b
     for dataset_name in sgs_by_dataset_for_cohort.keys():
         # If the dataset_name string ends in -test, strip test from the end
         # This is to ensure compatibility with the structure of our storage configs.
```

Comment:
Also remove this comment to revert the change.

Reply:
oops